### PR TITLE
Restyle the transaction panel.

### PR DESCRIPTION
**Pull request by jamescammarano**
Screenshot 2022-11-28 at 12 15 10 PM

### CODE REVIEW

File: src/rs_quark_assets/components/TransactionsPanel.vue [hunk 1]
```diff
@@ -134,6 +127,14 @@ onMounted(() => {
...
 const isLoading = computed(() => status?.last?.value?.status === STATUS.LOADING)
 
+function timestampFormatter(timestamp) {
+  const dateTime = new Date(timestamp)
+  const date = dateTime.toDateString().split(" ")
+  const time = dateTime.toTimeString().substring(0, 5)
+
+  return `${date[2]} ${date[1]}, ${time}`
+}
+
```
Comment (danielterwiel) [inline src/rs_quark_assets/components/TransactionsPanel.vue:137]: suggestion: I'd use date-fns format() for such operations. See Transactions.vue for an example.

Comment (danielterwiel) [reply src/rs_quark_assets/components/TransactionsPanel.vue:137]: If we were to use native js for date handling we should use Temporal, but last time I checked it didn't have satisfactory support yet.

Main goal is that we should support proper date display based on the user's locale. Both in format as well as adjustment for the user's timezone.

File: src/rs_quark_assets/components/TransactionsPanel.vue [hunk 2]
```diff
@@ -1,92 +1,84 @@
...
-      v-if="isLoading"
-      class="grid h-14 place-items-center"
-      aria-live="polite"
-      aria-atomic="true"
-      aria-busy="true"
-    >
-      <div class="flex gap-2">
-        <SvgSpinner class="h-6 w-6 animate-spin" />
-        Fetching resources
+  <div class="flex">
```
Comment (danielterwiel) [inline src/rs_quark_assets/components/TransactionsPanel.vue:2]: question: where do we need this wrapping element for?

File: src/rs_quark_assets/components/TransactionsPanel.vue (hunk 1 above)
Comment (jamescammarano) [reply src/rs_quark_assets/components/TransactionsPanel.vue:137]: That makes total sense. I'll look into it more.

File: src/rs_quark_assets/components/TransactionsPanel.vue [hunk 3]
```diff
@@ -97,30 +87,32 @@ export default {
...
 const { onCallResponse, status, execute } = quark.getPagedAccountTransactions(
   0,
   3,
   false,
 )
+
 onCallResponse(r => {
-  transactions.value = r.values
+  transactions.value = r.values.map(tr => {
+    tr.timestamp = format(new Date(tr.timestamp), "do MMM, HH:mm")
```
Comment (danielterwiel) [inline src/rs_quark_assets/components/TransactionsPanel.vue:113]: Suggestion: add user locale to format options. You can use the `get-user-locale` package

**Pull request merged**